client/src/assets/stylesheets/partials/_lesson.scss

```diff
@@ -10,6 +10,12 @@ $letterSize: 8em;
   padding: 2em 0;
 }
 
+.lesson-progress {
```

Comment:
1) Better name `.lesson-progress-bar`

2) Avoid using fixed `px` values at all costs since it isn't scalable with font scaling/zoom.  Use `rem` here.